rpcs3/Emu/Cell/SPULLVMRecompiler.cpp

```diff
@@ -6102,6 +6084,15 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 			return bitcast<f32[4]>((b & 0xff800000u) | (bitcast<u32[4]>(fpcast<f32[4]>(bnew)) & ~0xff800000u)); // Inject old sign and exponent
 		});
 
+		const auto [a, b] = get_vrs<f32[4]>(op.ra, op.rb);
+
+		if (g_cfg.core.spu_xfloat_accuracy == xfloat_accuracy::accurate)
+		{
+			const auto r = eval(fi(a, b));
```

Comment:
The reason it was commented out is because of the expendive transition between 512 simd and 256 simd.